tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1151,10 +1174,9 @@ def test_errored_sessions(self):
                     "timestamp": user_ts,
                     "tags": {
                         self.session_status_tag: indexer.record(org_id, "init"),
-                        self.release_tag: indexer.record(org_id, "foo"),
                     },
                     "type": "c",
-                    "value": 10,
+                    "value": 15,
```

Comment:
Why 15? Couldn't the `init` count still be 10, when all of them have errors?

Reply:
Yeah no particular reason.. I just wanted to make sure that there is a count that is healthy